src/Renderer/CsvRenderer.php

```diff
@@ -26,32 +11,61 @@
  */
 class CsvRenderer extends BaseRenderer
 {
-    private $stringBuffer;
 
+    /**
+     * @author Ayrton Ricardo<ayrton@voxtecnologia.com.br>
+     * @return mixed
+     */
     public function getStringBuffer()
     {
         return $this->stringBuffer;
```

Comment:
`stringBuffer` is a protected property on `BaseRenderer`. I forgot of remove the method `getStringBuffer` from `CsvRenderer`